r/tests/testthat/test-dataset.R

```diff
@@ -391,6 +391,17 @@ test_that("filter() with %in%", {
       collect(),
     tibble(int = df1$int[c(3, 4, 6)], part = 1)
   )
+
+  ds <- open_dataset(hive_dir, partitioning = hive_partition(other = utf8(), group = uint8()))
```

Comment:
It's hive-style so you don't need to tell it what the partitions are.

```suggestion
  ds <- open_dataset(hive_dir)
```